microsoft-edge/devtools-guide-chromium/whats-new/2022/07/devtools-104.md

```diff
@@ -0,0 +1,132 @@
+
+![Detached Elements List, Table, 5 rows, 3 columns](./devtools-104-images/a11y-detached-elements.png)
+<!-- Instructions for screenshot
+todo: see/use screenshot in work item attachment
+1. Navigate to: https://microsoftedge.github.io/Demos/detached-elements/
+2. Open DevTools > Detached Elements
+3. Send 5 messages in Room 1 by selecting the "Send 1 message" button 5 times
+4. Switch to Room 2
+5. In the Detached Elements tool, press the refresh arrow icon to get detached elements
+6. Take the screenshot and draw a red highlight box over the table (5 rows, 3 columns) in the Detached Elements tool -->
+
+<!-- Video recording of feature in action
+todo: see work item -->
+
+See also:
+* [Debug DOM memory leaks with the Detached Elements tool](../../../memory-problems/dom-leaks.md)
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+
+
+<!-- ====================================================================== -->
+## Fix: Arrows representing web socket traffic now render correctly in the Network tool
+<!-- rank: 4 -->
+
+<!-- todo: check work item eg video -->
+
+<!-- Title: Use the Network tool to understand web socket traffic -->
+<!-- Subtitle: Arrows representing messages from the server or client now display correctly in the Network tool. -->
+
+In previous versions of Microsoft Edge, when inspecting web socket traffic in the Network tool, the arrows representing whether a message was sent from the client or server were not rendering correctly.  In Microsoft Edge 104, this issue has been fixed.  Green up-arrows represent messages sent by the client, and red down-arrows represent messages sent by the server.  Thank you for sending us your feedback about this issue!
+
+![Network tool](./devtools-104-images/network-tool-ws-arrows.png)
+<!-- Start by using the work item attached png
+
+1. Navigate to: Online WebSocket & Socket.io Tester - PieSocket Realtime
+2. Open DevTools > Network and undock the tools
+3. On the website, select Connect and send a test message
+4. In the Network tool, select WS to filter to only WebSocket network requests
+5. Select the one network request that should appear
+6. Take a screenshot and draw a red highlight box over the red and green arrows -->
+
+See also:
+* [Network features reference](../../../network/reference.md)
+
+
+<!-- ====================================================================== -->
+## Announcements from the Chromium project
+
+Microsoft Edge 104 also includes the following updates from the Chromium project:
+
+* [Restart frame during debugging](https://developer.chrome.com/blog/new-in-devtools-104/#restart-frame)
+* [Slow replay options in the Recorder panel](https://developer.chrome.com/blog/new-in-devtools-104/#recorder)
+* [Build an extension for the Recorder panel](https://developer.chrome.com/blog/new-in-devtools-104/#recorder-extension)
```

Comment:
The recorder panel exist only in Chrome. It is not part of the Chromium open source project, and hence, not available in Edge. So I don't think we should include it in this Chromium project section.

Reply:
Has this been resolved yet @mikehoffms?

Reply:
@erdraud - yes, removed those two links.  closing thread as Resolved.  
Rendered: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/07/devtools-104?branch=pr-en-us-2072#announcements-from-the-chromium-project